great_expectations/data_context/store/store_backend.py

```diff
@@ -464,42 +406,40 @@ def _convert_key_to_filepath(self, key):
         self._validate_key(key)
 
         converted_string = self.filepath_template.format(*list(key), **{
-            "file_extension" : self.file_extension,
-            "file_prefix" : self.file_prefix,
+            "file_extension": self.file_extension,
```

Comment:
I've deprecated these in my branch, because they're redundant against `filepath_template`.

Assuming we want to keep that change, we'll have a small merge conflict to resolve.

Reply:
Let's resolve when merging the PR so that I don't reimplement it.